### DISCUSSION TITLE
error when run docker compose

### DISCUSSION THREAD

**Discussion by EngAly** [mockoon/mockoon#994] [Help]
I have this docker-compose file content

version: "3"
services:
  mockoon:
    image: mockoon/cli:latest
    container_name: mockoon
    ports:
      - "3001:3001"
    command: mockoon-cli start -d https://raw.githubusercontent.com/mockoon/mock-samples/main/samples/generate-mock-data.json

when i run this as docker-compose this error is appear

mockoon  |  ›   Error: Unexpected arguments: mockoon-cli, start
mockoon  |  ›   See more help with --help
mockoon exited with code 2

Operating system: Linux

Please any help will be appreciated

**Comment by 255kb** [comment]
This was answered before: https://github.com/mockoon/mockoon/discussions/955#discussioncomment-5082044
There is an [`ENTRYPOINT` in our Docker image](https://github.com/mockoon/mockoon/blob/main/packages/cli/docker/Dockerfile#L16), so you need to either override it or remove `mockoon-cli start` from your command.
